tests/integration/orcid/test_orcid_converter.py

```diff
@@ -387,3 +390,34 @@ def test_format_thesis_with_author_orcid(app, api_client):
     assert valid_against_schema(result)
     assert xml_compare(expected, result)
     assert _OrcidHasher(mock.Mock())._hash_xml_element(expected) == _OrcidHasher(mock.Mock())._hash_xml_element(result)
+
+
+@pytest.mark.usefixtures('isolated_app')
+class TestBibtexCitation(object):
+    def setup(self):
+        factory = TestRecordMetadata.create_from_file(__name__, 'test_orcid_converter_TestBibtexCitation.json')
+        self.inspire_record = factory.inspire_record
+
+    def test_do_add_bibtex_citation(self):
+        converter = OrcidConverter(
+            record=self.inspire_record,
+            url_pattern='http://inspirehep.net/record/{recid}',
+        )
+        xml_root = converter.get_xml(do_add_bibtex_citation=True)
+        found = False
+        for xml_element in xml_root.getchildren():
+            if 'citation' in xml_element.tag:
```

Comment:
Maybe this would fail in a more understandable way, and you won't need the description with the assert? Also no risk if we ever add a tag with 'citation' in it, but not being exactly equal.

```python
top_level_tags = [etree.QName(node).localname for node in xml_root.getchildren()]
assert 'citation' in top_level_tags
```